### PR TITLE
Add shoe warp

### PR DESCRIPTION
**Pull request by Splendide-Imaginarius**
Useful for tweaking the dimensions/shape of shoes.

Only accessible from Maker GUI for now.

### CODE REVIEW

File: Stiletto/HeelInfo.cs [hunk 1]
```diff
@@ -156,5 +228,93 @@ private void LateUpdate()
                 PostUpdate();
             }
         }
+
+        private void UpdateShoeWarp()
```
Comment (ManlyMarco) [inline Stiletto/HeelInfo.cs:269]: This is fairly expensive to run on every frame. At the very least check if user changed any of the transform properties and don't run this code if not.

**Review by ManlyMarco — CHANGES_REQUESTED** [review]
The code looks good to me as far as functionality goes, although it looks like performance can be a problem.

File: Stiletto/HeelInfo.cs (hunk 1 above)
Comment (Splendide-Imaginarius) [reply Stiletto/HeelInfo.cs:269]: Good point, thanks for noticing that. I'll see if I can optimize that a bit.

Comment (Splendide-Imaginarius) [reply Stiletto/HeelInfo.cs:269]: Hopefully better now?

**Review by ManlyMarco — APPROVED** [review]

File: Stiletto/HeelInfo.cs (hunk 1 above)
Comment (ManlyMarco) [reply Stiletto/HeelInfo.cs:269]: Yeah, that works, thanks.

**Pull request merged**